packages/core/admin/admin/src/content-manager/components/BlocksInput/BlocksToolbar.tsx

```diff
@@ -418,14 +353,48 @@ const ListButton = ({ block, format }: ListButtonProps) => {
     return false;
   };
 
+  const toggleList = (format: Block<'list'>['format']) => {
+    let currentListEntry;
+    if (editor.selection)
+      currentListEntry = Editor.above(editor, {
+        match: (node) => !Editor.isEditor(node) && node.type === 'list',
+      });
```

Comment:
please use a block of curly brackets

Reply:
I agree, is there a lint rule we can add for this?

Reply:
https://eslint.org/docs/latest/rules/curly